src/main/java/com/github/markusbernhardt/seleniumlibrary/keywords/RobotString.java

```diff
@@ -0,0 +1,37 @@
+package com.github.markusbernhardt.seleniumlibrary.keywords;
+
+import com.github.markusbernhardt.seleniumlibrary.RunOnFailureKeywordsAdapter;
+import org.apache.commons.lang3.StringUtils;
+import org.robotframework.javalib.annotation.ArgumentNames;
+import org.robotframework.javalib.annotation.Autowired;
+import org.robotframework.javalib.annotation.RobotKeyword;
+import org.robotframework.javalib.annotation.RobotKeywords;
+
+@RobotKeywords
+public class RobotString extends RunOnFailureKeywordsAdapter {
+
+    /**
+     * Instantiated Logging keyword bean
+     */
+    @Autowired
+    protected Logging logging;
+
+    @RobotKeyword("Set System Property")
+    @ArgumentNames({"key", "value"})
+    public void setSystemProperty(String key, String value) {
+        System.setProperty(key, value);
+    }
+
+    @RobotKeyword("Get System Property")
+    @ArgumentNames({"key"})
+    public String getSystemProperty(String key) {
+        return System.getProperty(key);
+    }
+
+    @RobotKeyword("Is Contain String Ignore Case")
+    @ArgumentNames({"str", "searchStr"})
+    public boolean isContainStringIgnoreCase(String str, String searchStr) {
```

Comment:
Isn't this duplicate witn https://robotframework.org/robotframework/latest/libraries/BuiltIn.html#Should%20Contain?

Also, if it's useful to redefine, same naming should be used. E.g. "Should Contain With Case Ignored"

Reply:
I agree with you)

Deleted